content/en/logs/archives/rehydrating.md

```diff
@@ -20,21 +20,23 @@ Index exclusion filters do not apply to historical views, so there is no need to
 
 ### Add new historical views
 
-1. **Select the archive** from which you wish to rehydrate log events. Only archives that are [configured to use role delegation](#permissions) are available for rehydrating.
+1. **Choose the time period** for which you wish to rehydrate log events.
 
-2. **Choose the time period** for which you wish to rehydrate log events.
+2. **Input the query**. The query syntax is the same as that of the [log explorer search][4], but is limited to log attributes, [reserved attributes][5], and free text search on the message.
```

Comment:
The reserved attributes link takes you to the logs overview page, is this the correct link?

Reply:
nice catch! no it's not. Some link lost in translation. I fixed it.